sonoshell/sonoshell.yaml

```diff
@@ -0,0 +1,14 @@
+sonobuoy-config:
+  driver: DaemonSet
+  plugin-name: sonoshell
+  result-format: manual
+  result-files:
+  - sonoshell.yaml
+spec:
```

Comment:
Add the command with the proper script invocation here:

```
spec:
  command:
  - ./sonoshell
  args:
  - "$(SONOBUOY_CONFIG_DIR)/test.yaml"
```

The $SONOBUOY_CONFIG_DIR is an env var we add at runtime and is the directory where the configmaps will be mounted.